### DISCUSSION TITLE
Sort Table Rows?

### DISCUSSION THREAD

**Discussion by infoseckev** [Textualize/rich#1434] [Q&A] [answered]
It seems `add_row() ` will add data below the last row in the Table. I would like to be able to sort this information, move rows around. 

Let's say I'm working on a UI similar to airmon-ng, where I always want the most recent Row to appear on top of a Table and also be able to delete certain rows.

Is there a way to get the current rows from a Table so I can manipulate the data, and then re-insert it inside the Table?

**Comment by willmcgugan** [comment, marked as answer]
Think of a Rich table as an intermediate stage between your data and the terminal. Its only job is to render to the terminal. If you want to sort the the rows, you can rebuild a Table object however you like.
